lighthouse-core/gather/gatherers/dobetterweb/optimized-images.js

```diff
@@ -82,9 +82,11 @@ class OptimizedImages extends Gatherer {
       const isSameOrigin = URL.hostsMatch(pageUrl, record._url);
```

Comment:
does this need to be looking at origins instead of just host so the protocol is included?